.github/workflows/ci_mac.yml

```
@@ -199,6 +199,19 @@ jobs:
           exit $rc
         fi
 
+    - name: Testing GUI
+      if: matrix.skip_testing != true
+      working-directory: tests
+      run: |
+        export ASAN_OPTIONS=fast_unwind_on_malloc=0
+        # Exclude tests that are currently known to fail
+        export wx_tests_selection='~[wxTextCtrl] ~[item-container] ~[persist] ~[html] ~[font] ~[wxWebView] ~[valnum] ~[radiobutton] ~[clip] ~wxExecute::RedirectUTF8 ~Ellipsization::NormalCase ~SpinCtrl::Wrap ~SpinCtrlDouble::Arrows ~SpinCtrlDouble::Wrap ~SpinCtrl::Arrows ~SpinCtrlDouble::Increment ~Grid::KeyboardSelection ~NotebookTestCase ~TextCtrlTestCase ~SettingsTestCase ~XRC::LoadURL'
```

Pretty minor, but we don't need to export this
```suggestion
        wx_tests_selection='~[wxTextCtrl] ~[item-container] ~[persist] ~[html] ~[font] ~[wxWebView] ~[valnum] ~[radiobutton] ~[clip] ~wxExecute::RedirectUTF8 ~Ellipsization::NormalCase ~SpinCtrl::Wrap ~SpinCtrlDouble::Arrows ~SpinCtrlDouble::Wrap ~SpinCtrl::Arrows ~SpinCtrlDouble::Increment ~Grid::KeyboardSelection ~NotebookTestCase ~TextCtrlTestCase ~SettingsTestCase ~XRC::LoadURL'
```